src/components/AircraftPopup.jsx

```diff
@@ -112,35 +89,31 @@ export default observer(function AircraftPopup(properties) {
       focusAfterOpen={false}
       onClose={() => cwpStore.closeLevelPopupForAircraft(aircraftId)}
     >
-      <DraggableCore onStart={onDragStart} onDrag={onDrag}>
-        <div>
-          <Button size="sm" variant="dark" onClick={() => cwpStore.closePopupForAircraft(aircraftId)}>x</Button>
-          <Container className="flight-popup-container">
-            <Row>
-              <Col className="gutter-2" onClick={() => setController()}>{callSign}</Col>
-            </Row>
-            <Row>
-              <Col className="gutter-2" onClick={() => cwpStore.openLevelPopupForAircraft(aircraftId)}>{Number.parseFloat((altitude).toFixed(0))}</Col>
-              <Col className="gutter-2" onClick={() => cwpStore.toggleFlightRouteForAircraft(aircraftId)}>
-                {nextFix}
-              </Col>
-              <Col className="gutter-2" />
-            </Row>
-            <Row>
-              <Col className="gutter-2" onClick={() => cwpStore.toggleSpeedVectorForAircraft(aircraftId)}>{speedAndWakeTurbulenceLabel}</Col>
-              <Col className="gutter-2" onClick={() => cwpStore.openLevelPopupForAircraft(aircraftId)}>NSFL</Col>
-              <Col className="gutter-2" />
-            </Row>
-            <Row>
-              <Col className="gutter-2" onClick={() => cwpStore.openNextSectorPopupForAircraft(aircraftId)}>{nextSectorController}</Col>
-              <Col className="gutter-2">{assignedFlightLevel}</Col>
-              <Col className="gutter-2" onClick={() => cwpStore.openLevelPopupForAircraft(aircraftId)}>COO</Col>
-            </Row>
-          </Container>
-
-        </div>
-      </DraggableCore>
-    </Popup>
-
+      <div>
+        <Button size="sm" variant="dark" onClick={() => !isDragging() && cwpStore.closePopupForAircraft(aircraftId)}>x</Button>
+        <Container className="flight-popup-container">
+          <Row>
+            <Col className="gutter-2" onClick={() => !isDragging() && setController()}>{callSign}</Col>
```

Comment:
I added the `!isDragging() && ` in front of the method calls.

The idea is to disable click events when the dragging occurs, but in Web it's actually difficult to disable events on children from parents, because the events are "bubbling up", or starting from the children to their parents, recursively.

In other words I did a small hack to be able to both click and drag the popup by clicking its buttons. I think is not too bad. It's not very "react" but that will do.